src/passes/RemoveUnusedBrs.cpp

```diff
@@ -521,7 +524,8 @@ struct RemoveUnusedBrs : public WalkerPass<PostWalker<RemoveUnusedBrs>> {
         return false;
       }
       // if there is control flow, we must stop looking
-      if (EffectAnalyzer(getPassOptions(), curr).branches) {
+      if (EffectAnalyzer(getPassOptions(), getModule()->features, curr)
+            .branches) {
```

Comment:
this is an example of why I think merging `mayThrow` into `branches` makes sense - this location cares about `mayThrow` as well, I think.

Reply:
Forgot to answer this. This has been changed to `transfersControlFlow` to take exceptions into account.